prow/cmd/branchprotector/README.md

```diff
@@ -183,44 +181,20 @@ This will say how the binary will actually change github if you add a
 
 Run things like the following:
 ```sh
-# Build and push image, create job
-bazel run //prow/cmd/branchprotector:oneshot.create
-# Delete finished job
-bazel run //prow/cmd/branchprotector:oneshot.delete
-# Describe current state of job
-bazel run //prow/cmd/branchprotector:oneshot.describe
+# Build image locally
+make -C prow push-single-image PROW_IMAGES=prow/cmd/branchprotector REGISTRY=<YOUR_REGISTRY>
 ```
 
-This will build an image with your local changes, push it to
-`STABLE_DOCKER_REPO` and then deploy to `STABLE_PROW_CLUSTER`.
-
-See [`print-workspace-status.sh`] for the definition of these variables.
-See [`oneshot-job.yaml`] for details on the deployed job resource.
+This will build an image with your local changes, push it to `<YOUR_REGISTRY>`
 
 ### Deploy cronjob to production
 
-Follow the standard prow deployment process:
-
-```sh
-# build and push image, update tag used in production
-prow/bump.sh branchprotector
-# apply changes to production
-bazel run //config/prow/cluster:branchprotector.apply
-```
-
-See [`prow/bump.sh`] for details on this script.
-See [`config/prow/cluster/branchprotector_cronjob.yaml`] for details on the deployed
-job resource.
-
+[branchprotector image]() is automatically built and bumped as part of prow, see
+[build_test_update.md#how-to-update-the-cluster](/prow/build_test_update.md#how-to-update-the-cluster)
+for more details.
```

Comment:
We do not include a branchprotector cronjob as part of deploying to the cluster. I think we now use either a periodic or postsubmit ProwJob.

Reply:
good point, updated